### DISCUSSION TITLE
How to make  arrow centered when the placement is right-start

### DISCUSSION THREAD

**Discussion by BasmaNabil** [floating-ui/floating-ui#2050] [Q&A]
I have a popup and i want to make the arrow centered when the placement is right-start
`
        <FloatingPortal>
              {state.open && (
                <div
                  ref={ref}
                  className="popup"
                  style={{
                    position: state.strategy,
                    top: state.y ?? 0,
                    left: state.x ?? 0,
                    visibility: state.x == null ? "hidden" : "visible",
                  }}
                  aria-describedby={`${state.id}-desc`}
                >
                  {props.children}
                  <div
                    ref={state.arrowRef}
                    className="popup-arrow"
                    style={{
                      position: "absolute",
                      width: "50px",
                      height: "50px",
                      left: arrowX != null ? `${arrowX}px` : "",
                      top: arrowY != null ? `${arrowY}px` : "",
                      [staticSide]: "-5px",
                      transform: "rotate(45deg)",
                    }}
                  />
                </div>
              )}
            </FloatingPortal>
`

**Comment by atomiks** [comment]
Do you mean centered in itself, or centered to the reference element? The latter already happens with the dynamic axis positioning as you have (although, really hard to read, please try and format it properly), former you can use a static 50% with neg margin/translation
